<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WindowNavigatorClass</class>
 <widget class="QMainWindow" name="WindowNavigatorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>91</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Navigator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pbMain">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Main</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRecon">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reconstruction</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbOut">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Output</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbLRF">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>LRF</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbGeometry">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Geometry</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbGraph">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Graph</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbMaterials">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>160</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Materials</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>224</y>
      <width>118</width>
      <height>3</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pbMinAll">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>227</y>
      <width>46</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Hide</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbMaxAll">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>227</y>
      <width>46</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Show</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="pb">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>251</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="textVisible">
     <bool>true</bool>
    </property>
    <property name="invertedAppearance">
     <bool>false</bool>
    </property>
    <property name="textDirection">
     <enum>QProgressBar::TopToBottom</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pbExamples">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Examples</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbScript">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Script</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbNewLRF">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>New LRF</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbDetector">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Detector</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
